spectrum-offchain-cardano/src/data/pool.rs

```diff
@@ -76,6 +86,9 @@ impl<Order> ApplyOrderError<Order> {
                 ApplyOrderError::LowBatcherFee(low_batcher_fee.map(f))
             }
             ApplyOrderError::Incompatible(math_error) => ApplyOrderError::Incompatible(math_error.map(f)),
+            ApplyOrderError::VerificationFailed(verification_error) => {
```

Comment:
can we verify this in advance, so we don't have to find out during execution?

Reply:
E.g. in TryFromLedger

Reply:
nope, signature verification requires `pool_nonce` field